CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [minor]
+### Changed
+- SFX-249: Created tracker service plugin for Search
+  - Added `origin` to:
+    - `SaytProductsRequestPayload`
+    - `SearchRequestPayload`
+    - `UpdateSearchTermPayload`
+
+### Added
+- SFX-249: Created tracker service plugin for Search
+  - Event name constants:
+    - `BEACON_SEARCH`
+  - Event payload interfaces:
+    - `BeaconSearchPayload`
+  - Dependencies:
+    - `gb-tracker-client`
```

Comment:
In my opinion, the descriptions here should be more tightly scoped to this repo/package. If you took this package in isolation, does it all still make sense? Without the greater context, I don't think mentioning a "tracker plugin" in the events repo really makes sense.

Perhaps you can go with something like this?

```markdown
### Changed
- ELE-249: Added a required `origin` property to the following interfaces:
  - `SaytProductsRequestPayload`
  - `SearchRequestPayload`
  - `UpdateSearchTermPayload`

### Added
- ELE-249: Added search beacon-related event.
  - Event name constant: `BEACON_SEARCH`
  - Event payload interface: `BeaconSearchPayload`
- ELE-249: Added `gb-tracker-client` as a dependency.
```

As well, our tickets are `ELE` now, not `SFX`.

Reply:
I'm the worst. I'll remember the ELE switch eventually. And yes, you're right about the lack of context.